CGBEmu/source/Player.h

```diff
@@ -1,24 +1,14 @@
 #include <iostream>
 class Player
 {
-	int width;
-	int height;
 public:
 	Player();
 	~Player();
-	void set_values();
-	int area() { return width*height; }
-};
 
-Player::~Player()
```

Comment:
notice that all function implementations(definitions) have been moved to the Player.cpp file